src/pages/snap/{Snap.location}/{Snap.slug}.test.tsx

```diff
@@ -8,16 +9,47 @@ import {
   getMockSnap,
 } from '../../../utils/test-utils';
 
+jest.mock('wagmi', () => ({
+  useAccount: jest.fn(),
+}));
+
 describe('Snap page', () => {
+  let mockUseAccount: jest.Mock;
+
+  beforeEach(() => {
+    mockUseAccount = useAccount as jest.Mock;
+    mockUseAccount.mockClear();
+  });
+
   it('renders', async () => {
+    mockUseAccount.mockReturnValue({ isConnected: true });
```

Comment:
nic:

add space after mock, apply to all below

Reply:
have changed all area with mockUserAccount with space between mockUseAccount and actual test.